### PR TITLE
Atlantic 2024 schedule

### PR DESCRIPTION
**Pull request by plaindocs**
and related post

- [x] Schedule approved by speakers

----
📚 Documentation preview 📚: https://writethedocs-www--2177.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Hey everyone!
+
+We're around two months away from the next online {{city}} conference, and we're working hard to make sure it'll be as welcoming and fun as previous years.
+
+Today we're announcing the full presentation schedule.
+
+Conference platform
+--------------------
+
+The conference will again be held on Venueless this year, so some things might looks a little different.
```
Comment (hillaryfraley) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
The conference will again be held on Venueless this year, so some things might look a little different.
```

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+**All talk videos will be published on our YouTube channel no later than 1 week after the conference.**
+
+Start thinking about Writing Day
+--------------------------------
+
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
+
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page `submit your project <https://forms.gle/KPo1ZPuRHqf7UZy37>`_ and we'll
```
Comment (hillaryfraley) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
If you'd like us to include your project on the Writing Day page, `submit your project <https://forms.gle/KPo1ZPuRHqf7UZy37>`_ and we'll
```

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+--------------------
+
+The `Write the Docs {{city}} {{year}} Pop-Up Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
+
+The campaign will run in batches until October 31st. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
+
+All our events and activities
+-----------------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.
```
Comment (hillaryfraley) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this year's conference is held online, we're holding to that plan.
```

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
```
Comment (hillaryfraley) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/lightning-talks/>`__ each day after lunch. These are 5-minute presentations and a lot of fun. They can be about anything you want; this is your chance to share something you are passionate about with the audience. Slides are not required.
```

**Review by mxsasha — APPROVED** [review]
Looks good, I've taken the liberty to make some commits directly to the post :)

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+Our updated Welcome Wagon guide, available soon, will help it feel familiar before you've even arrived.
+
+All ticket holders will receive an invite to Venuless around one week before the conference.
+If you don't have your ticket yet, `register one soon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_, as we may sell out.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`/conf/{{shortcode}}/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference.
+We'll continue to hold live Q&A sessions after most talks. The Q&A will come from your questions but moderated by our MCs.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
We'll continue to hold live Q&A sessions after most talks. The Q&A will come from your questions but be moderated by our MCs.
```

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page, `submit your project <https://forms.gle/j44pUr31RNyzy5pL6>`_ and we'll
+add it to our `Writing Day project list <https://www.writethedocs.org/conf/atlantic/{{year}}/writing-day/#your-project-here>`__.
+You can also just show up on Sunday ready to introduce your project, but getting it on the website in advance can increase interest.
+
+
+Shirts still on sale
+--------------------
+
+The `Write the Docs {{city}} {{year}} Pop-Up Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst:53]: Is it open? The link is empty for me.

Comment (plaindocs) [reply docs/conf/atlantic/2024/news/announcing-schedule.rst:53]: Waiting on @thatdocslady for this. We can always bounce it to next announcement?

**Comment by plaindocs** [comment]
Nice one folks.

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+Conference platform
+--------------------
+
+This year, the conference will be held on a new platform, Venueless, so some things will look a little different.
+But you'll find the usual spaces and experiences anyhow, including a stage for our amazing speakers,
+integrated chat, unconference rooms with breakout calls, and expo booths.
+
+Our updated Welcome Wagon guide, available soon, will help it feel familiar before you've even arrived.
+
+All ticket holders will receive an invite to Venuless around one week before the conference.
```
Comment (CollierCZ) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
All ticket holders will receive an invite to Venueless around one week before the conference.
```

File: docs/conf/atlantic/2024/news/announcing-schedule.rst (hunk 6 above)
Comment (mxsasha) [reply docs/conf/atlantic/2024/news/announcing-schedule.rst:53]: The shop is ready :)

File: docs/conf/atlantic/2024/news/announcing-schedule.rst [hunk 8]
```diff
@@ -0,0 +1,83 @@
+:template: {{year}}/generic.html
+
+.. post:: July 22, 2024
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/news/announcing-schedule.rst]:
```suggestion
.. post:: July 30, 2024
```

**Pull request merged**